src/parameterizations/lateral/MOM_mixed_layer_restrat.F90

```diff
@@ -98,11 +99,15 @@ module MOM_mixed_layer_restrat
   real    :: Kv_restrat            !< A viscosity that sets a floor on the momentum mixing rate
                                    !! during restratification, rescaled into thickness-based
                                    !! units [H2 T-1 ~> m2 s-1 or kg2 m-4 s-1]
+  logical :: MLD_grid              !< If true, read a spacially varying field for MLD_decaying_Tfilt
+  logical :: Cr_grid               !< If true, read a spacially varying field for Cr
 
   real, dimension(:,:), allocatable :: &
          MLD_filtered, &           !< Time-filtered MLD [H ~> m or kg m-2]
          MLD_filtered_slow, &      !< Slower time-filtered MLD [H ~> m or kg m-2]
-         wpup_filtered             !< Time-filtered vertical momentum flux [H L T-2 ~> m2 s-2 or kg m-1 s-2]
+         wpup_filtered, &          !< Time-filtered vertical momentum flux [H L T-2 ~> m2 s-2 or kg m-1 s-2]
+         MLD_filtered_space, &     !< Spatially varying time scale for MLD filter [T ~> s]
```

Comment:
The name `MLD_filtered_space` would seem to describe a depth, and not a time-scale. For me, this would be clearer if it were renamed as `MLD_Tfilt_space` or something similar that hints that this is a time-scale.